TgBot.java

```diff
@@ -0,0 +1,156 @@
+    }
+
+    public synchronized void firstKeyboard(String chatId, String msg){
+
+        ReplyKeyboardMarkup keyboardMarkup = new ReplyKeyboardMarkup();
+        ArrayList<KeyboardRow> firstKeyboard = new ArrayList<>();
+        KeyboardRow row1 = new KeyboardRow();
+        KeyboardRow row2 = new KeyboardRow();
+        row1.add("Go Shopping");
+        row2.add("Shop's Info");
+        firstKeyboard.add(row1);
+        firstKeyboard.add(row2);
+
+        keyboardMarkup.setResizeKeyboard(true);
+        keyboardMarkup.setKeyboard(firstKeyboard);
+        keyboardMarkup.setOneTimeKeyboard(false);
+
+        SendMessage message = new SendMessage();
+        message.setChatId(chatId);
+        message.setReplyMarkup(keyboardMarkup);
+
+        switch (msg) {
+            case "/start", "Main Page" -> message.setText("Welcome to our shop!");
+            case "Shop's Info" -> message.setText("Info");
+        }
+
+        try {
+            execute(message);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    long pageCounter = 1;
+
+    public synchronized void keyboard(String chatId, String msg){
+
+        ReplyKeyboardMarkup keyboardMarkup = new ReplyKeyboardMarkup();
+        ArrayList<KeyboardRow> keyboard = new ArrayList<>();
+        KeyboardRow row1 = new KeyboardRow();
+        KeyboardRow row2 = new KeyboardRow();
+        row1.add("Back");
+        row1.add(String.valueOf(pageCounter));
+        row1.add("Forward");
+        row2.add("Main Page");
+        keyboard.add(row1);
+        keyboard.add(row2);
+        keyboardMarkup.setResizeKeyboard(true);
+        keyboardMarkup.setKeyboard(keyboard);
+
+        if (msg.equals("Forward")){
+            pageCounter++;
+            row1.set(1, String.valueOf(pageCounter));
+        } else if (msg.equals("Back") && pageCounter > 1) {
+            pageCounter--;
+            row1.set(1, String.valueOf(pageCounter));
+        }
+
+        sendPhoto(chatId, pageCounter);
+
+        SendMessage message = new SendMessage();
+        message.setChatId(chatId);
+        message.setText("Product description...");
+        message.setAllowSendingWithoutReply(true);
+        message.setReplyMarkup(keyboardMarkup);
+
+        try {
+            execute(message);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+        inlineKeyboard(chatId);
+    }
+
+    public synchronized void inlineKeyboard(String chatId){
+        SendMessage sendMessage = new SendMessage();
+        sendMessage.setChatId(chatId);
+        sendMessage.setText("Add to cart?");
+
+        InlineKeyboardMarkup inlineKeyboardMarkup = new InlineKeyboardMarkup();
+        List<List<InlineKeyboardButton>> keyboard = new ArrayList<>();
+        List<InlineKeyboardButton> button1 = new ArrayList<>();
+        InlineKeyboardButton but = new InlineKeyboardButton();
+        but.setText("Add");
+        but.setCallbackData("Add to cart");

```

Comment:
Could you please go through the PR and unhardcode strings? It could be either a constant class (which is appreciable for the first team project) or a resource file.